geniza/corpus/models.py

```diff
@@ -449,11 +450,33 @@ def unidecode_tag(sender, instance, **kwargs):
         instance.name = unidecode(instance.name)
 
 
+class MetadataManager(models.Manager):
```

Comment:
Instead of a secondary manager, let's use a custom queryset with a method we can call when we want to apply this prefetching.  Maybe the method on the queryset could be something like `metadata_prefetch` ; or if there are multiple groups of prefetching we need in different places, we could have two different methods (they can stack).

Then we can replace the default manager (`Document.objects`) with one generated from our queryset: https://docs.djangoproject.com/en/3.2/topics/db/managers/#creating-a-manager-with-queryset-methods

Then in the admin or metadata export, we can do `Document.objects.metadata_prefetch()` or whatever you decide to call the method(s). And in admin, this method can be applied to the existing queryset passed in (can revert to previous logic where we call `super()` and don't have to apply any ordering).